software/generic/DeviceDescription.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef __DEVICE_DESCRIPTION_HPP__
+#define __DEVICE_DESCRIPTION_HPP__
+
+#include <Types.hpp>
+
+enum class DeviceType {
+  Wearable,
+  Bike,
+};
+
+class DeviceDescription {
+ public:
+  const DeviceType type;
+  const uint8_t physicalLeds;
```

Comment:
I've been doing a lot more C++ at work, and getting familiar with the Google style. Variable naming in C++ is typically snake_case - can you update the vars in this file to that style?

Reply:
How about function parameters?

Reply:
Those too, yeah